core/src/test/scala/org/apache/spark/benchmark/BenchmarkBase.scala

```diff
@@ -48,15 +48,11 @@ abstract class BenchmarkBase {
       if (!file.exists()) {
         file.createNewFile()
       }
-      output = Some(new FileOutputStream(file))
+      output = Option(new FileOutputStream(file))
```

Comment:
This looks like irrelevant pig-back.

Reply:
Change here because: https://github.com/apache/spark/pull/22443#discussion_r221181428

Reply:
IIUC, @HyukjinKwon meant `when you need to touch this file`.

Reply:
I am worried that I will forget it after a long time, so I am changing this time. I should revert it?

Reply:
Why do you replace `Some` to `Option`? Are you worrying `new FileOutputStream(file)` becomes `null`?

Reply:
My point was that there's no point of checking `null` below from my cursory look. If there's no chance that it becomes `null`, we can leave it `Some` and remove `null` check below.